### PR TITLE
Zen DNN - Docker & Tests

### PR DESCRIPTION
**Pull request by kiritigowda**
* Docker
  + CPU Only
  + GPU HIP

* Tests
  + Convolution Test
  + Inference - MNIST 
  + Inference - AlexNet

### CODE REVIEW

File: tests/zen_dnn_tests/conv/include/test_utils.hpp [hunk 1]
```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+* Modifications Copyright (c) 2021 Advanced Micro Devices, Inc. All rights reserved.
+* Notified per clause 4(b) of the license.
+*******************************************************************************/
+
+/*******************************************************************************
+* Copyright 2019 Intel Corporation
```
Comment (rrawther) [inline tests/zen_dnn_tests/conv/include/test_utils.hpp:7]: Why this is saying copy right to Intel

File: tests/zen_dnn_tests/conv/include/test_utils.hpp [hunk 2]
```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+* Modifications Copyright (c) 2021 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (rrawther) [inline tests/zen_dnn_tests/conv/include/test_utils.hpp:2]: FIx correct date

File: tests/zen_dnn_tests/mnist/include/test_utils.hpp [hunk 3]
```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+* Modifications Copyright (c) 2021 Advanced Micro Devices, Inc. All rights reserved.
+* Notified per clause 4(b) of the license.
+*******************************************************************************/
+
+/*******************************************************************************
+* Copyright 2019 Intel Corporation
```
Comment (rrawther) [inline tests/zen_dnn_tests/mnist/include/test_utils.hpp:7]: Copyright to Intel??

**Review by rrawther — CHANGES_REQUESTED** [review]
Do we need to push the binary data for mnist as well. I thought we have it somewhere else. Can we refer from there instead

**Review by LakshmiKumar23 — COMMENTED** [review]
@kiritigowda can we put all the images in MIVisionX/data? do we want the test images separately here?

**Comment by LakshmiKumar23** [comment]
@kiritigowda @rrawther Seems like the test_utils.h file is replicated in all examples, can we have it as a common header file?

**Comment by LakshmiKumar23** [comment]
@kiritigowda you deleted the .hpp file altogether

**Comment by kiritigowda** [comment]
@LakshmiKumar23 yes will use from zen dnn git source

**Pull request merged**